src/styles/blocks/card.scss

```
@@ -0,0 +1,90 @@
+.card {
+  width: $width-card;
+  padding: 32px 16px 16px;
+
+  border: 1px solid $border-card-colors;
+  border-radius: 5px;
+
+  &__logo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+  }
+
+  &__link {
+    font-family: Roboto, sans-serif;
+    font-size: $font-size-normal;
+    color: $main-color;
+
+    text-decoration: none;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-size: $font-size-small;
+    line-height: 14px;
+    font-weight: $font-weight-small;
+    color: $light-text-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__value {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    align-self: center;
+
+    font-size: $font-size-small;
+    line-height: 14px;
+    font-weight: $font-weight-small;
+  }
+
+  &__price {
+    font-weight: $font-weight-small;
+    color: $light-text-color;
+  }
+
+  &__cost {
+    font-size: $font-size-large;
+    font-weight: 700;
+  }
+
+  &__button {
+    display: flex;
+    width: 166px;
+    justify-content: center;
+
+    font-size: 14px;
+    line-height: 38px;
+    font-weight: $font-weight-large;
+    color: $white-color;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    background-color: $page-color;
+    border-radius: 5px;
+    border: 1px solid transparent;
+
+    transition: all 0.3s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```